### ISSUE TITLE
Sync multiple scripts

### ISSUE THREAD

**Issue by Mottie** [OpenUserJS/OpenUserJS.org#1134]
I updated a bunch of my userscripts that are all contained within one repository. As automatic syncing doesn't appear to work (#723), I was forced to manually update each one by using "Add Script" > "Import Script from GitHub" > choosing the repository, then clicking the "Import" button for all 25+ scripts.

Please add a method to manually sync all of our userscripts, or all of the userscripts in a particular repository.

**Comment by Martii** [comment]
Thanks for the suggestion. 

As you noted #723 would be a dependency on this for libraries. Regular Userscripts, ones that end in .user.js, do actually sync when S3 isn't having issues which is what I'm working on at the moment. The *github* dependency has been major semver idle at v9.x for a while so perhaps this could happen with one big chunk... but again it's a later thing for evaluation.

**Comment by Martii** [comment]
So this is first time in a while that I've had to resync most of my scripts... GH's caching of the raw url prevented me from bulk changing a bunch of my scripts. 

What this means is that even though I went to import each one manually after the fact the raw url was still serving an old version for a lot of them. So this is something that shouldn't be encouraged on OUJS since it is a bandwidth hog on our end and will probably end up using more bandwidth in general e.g. be patient with GH's caching mechanism... one step at a time.

Closing as won't fix.
